backend/planty/planty/settings.py

```diff
@@ -122,3 +152,7 @@
 # https://docs.djangoproject.com/en/4.0/ref/settings/#default-auto-field
 
 DEFAULT_AUTO_FIELD = 'django.db.models.BigAutoField'
+
+LOGIN_URL = '/users/login/'
```

Comment:
Czy to jest potrzebne dla rest api?

Reply:
To chyba zostawiłem po innym tutorialu :)
Trzeba przetestować czy to jakoś wpływa, ale podobnie nie powinno

Reply:
okej, w takim razie można merge